src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    public ColorSupplier(Random random) {
+        this.random = random;
```

Comment:
it's better to initialize this variable in the same place where you declare this.

Reply:
remove this constructor, we don't need this anymore